lnwallet/rpcwallet/rpcwallet.go

```diff
@@ -743,6 +743,8 @@ func (r *RPCKeyRing) remoteSign(tx *wire.MsgTx, signDesc *input.SignDescriptor,
 	// Assemble all other information about the input we have.
 	in.RedeemScript = sigScript
 	in.SighashType = signDesc.HashType
+	// TODO(roasbeef): dependent on finishing the PSBT work for taproot
```

Comment:
Went to go update the rpcwallet, and realized that since it's all PSBT based now, we need to address [#taproot psbt](https://github.com/lightningnetwork/lnd/issues/6268)

Reply:
Working on the PSBT implementation! Should be ready soon.